apps/web/src/components/tx/security/tenderly/utils.ts

```diff
@@ -106,3 +109,41 @@ export const getSimulationPayload = async (params: SimulationTxParams): Promise<
     save_if_fails: true,
   }
 }
+
+export const getCallTraceErrors = (simulation?: TenderlySimulation) => {
+  if (!simulation || !simulation.simulation.status) {
+    return []
+  }
+
+  return simulation.transaction.call_trace.filter((call) => call.error)
+}
+
+export type SimulationStatus = {
+  isLoading: boolean
+  isFinished: boolean
+  isSuccess: boolean
+  isCallTraceError: boolean
+  isError: boolean
+}
+
+export const getSimulationStatus = (simulation: UseSimulationReturn): SimulationStatus => {
```

Comment:
Can't this live in the utils package? It looks like something that we would also need in the mobile app.

Reply:
There is no issue moving it there.
But should we move it there when the mobile app starts using it? Or should I do it now?